src/elements/content-sidebar/__tests__/Sidebar.test.js

```diff
@@ -205,6 +206,12 @@ describe('elements/content-sidebar/Sidebar', () => {
                 expect(wrapper.hasClass('bcs-is-open')).toBe(expected);
             },
         );
+
+        test('should not render SidebarNav when hasNav is false', () => {
+            const wrapper = getWrapper({ hasNav: false });
+
+            expect(wrapper.containsMatchingElement(<SidebarNav />)).toBe(false);
```

Comment:
I've typically seen the usage of `wrapper.exists` vs `containsMatchingElement`. When should we use one versus the other?

Reply:
I've changed to using `wrapper.exists` to keep the tests more consistent.